ChangeLog

```diff
@@ -4,9 +4,15 @@ Changes in squid-6.4 (15 Oct 2023):
 	- Regression: Do not use static initialization to register modules
 	- Bug 5301: cachemgr.cgi not showing new manager interface URLs
 	- Bug 5300: cachemgr.cgi assertion
+	- Fix handling of large HTTP headers in storeClientCopy()
+	- Fix store_client caller leak on certain errors
+	- Fix URL percent-encoding
+	- Miss if an HTTP 304 update would exceed reply_header_max_size
 	- HTTP: Improve handling of empty lines received prior to request-line
 	- Y2038: improve printing of time settings
 	- Extend cache_log_message to problematic from-helper annotations
+	- Add some HTTP revalidation tests
```

Comment:
```suggestion
	- CI: More HTTP caching and revalidation tests
```

I do not insist on this change, but it is better to match the commit title prefix than to drop "caching" IMO. I also think it is fairly important to disclose that this change is about CI rather than some runtime HTTP revalidation tests in Squid code.